README

```diff
@@ -240,7 +245,7 @@ Features
 
     - netapp
       Plugin to query performance values from a NetApp storage system using the
-      “Manage ONTAP” SDK provided by NetApp.
+      “Manage ONTAP” SDK provided by NetApp.
```

Comment:
must have been fixed by my editor. no-break space symbol (c2a0) was changed to space symbol (20).
Should I revert this change?